### PR TITLE
Feature/ER-1148 display warning when revoking recruitment permission

### PR DESCRIPTION
**Pull request by sachpatel**
I know the changes made in this PR are not ideal and make the razor view code quite complex to look at but the conditional logic and displaying of conditional content is only for the Recruitment Provider private beta. I would hope to remove the conditional code ore provide a more appropriate way of showing content related to a particular permissions being toggled.

### CODE REVIEW

File: src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml [hunk 1]
```diff
@@ -40,35 +41,104 @@
...
+                                                    <label class="govuk-label govuk-radios__label" for="operation-@i-no">
+                                                        No
+                                                    </label>
+                                                </div>
                                             </div>
+                                        </fieldset>
+                                    </div>
+                                </td>
+                            </tr>
+                            if (isRecruitmentPermission && Model.Operations[i].IsEnabled)
```
Comment (chrisjensenuk) [inline src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml]: We've already checked for isRecruitmentPermission so don't need to do this again.

File: src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml [hunk 2]
```diff
@@ -40,35 +41,104 @@
...
+                                            </p>
+                                            <p>
+                                                All vacancies previously created by the provider will remain with them, to transfer vacancies back to you, contact us at <a href="mailto:VacancyServices.RAA@education.gov.uk">VacancyServices.RAA@education.gov.uk</a>
+                                            </p>
+                                        </div>
+                                    </td>
+                                </tr>
+                            }
+
+                            if (isRecruitmentPermission && Model.Operations[i].IsEnabled == false)
```
Comment (chrisjensenuk) [inline src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml]: ditto

Comment (sachpatel) [reply src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml]: 👍

File: src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml (hunk 1 above)
Comment (sachpatel) [reply src/SFA.DAS.ProviderRelationships.Web/Views/AccountProviderLegalEntities/Get.cshtml]: 👍

**Pull request merged**